src/data/set/finite.lean

```diff
@@ -269,12 +283,28 @@ by rw ← inter_eq_self_of_subset_right h; apply_instance
 theorem finite.subset {s : set α} : finite s → ∀ {t : set α}, t ⊆ s → finite t
 | ⟨hs⟩ t h := ⟨@set.fintype_subset _ _ _ hs (classical.dec_pred t) h⟩
 
+lemma finite.union_iff {s t : set α} : finite (s ∪ t) ↔ finite s ∧ finite t :=
+⟨λ h, ⟨h.subset (subset_union_left _ _), h.subset (subset_union_right _ _)⟩,
+ λ ⟨hs, ht⟩, hs.union ht⟩
+
+lemma finite.diff {s t u : set α} (hs : s.finite) (ht : t.finite) (h : u \ t ≤ s) : u.finite :=
```

Comment:
From the lemma name I'd expect the statement
```suggestion
lemma finite.diff {s : set α) (t : set α) (hs : s.finite) : (s \ t).finite :=
```
Assuming this is true, do we have it under another name?

Reply:
It is true, because it is a subset of s. And we don't have it under a different name. I agree that the lemma I need should probably be named differently. Any suggestions?